HopsanGUI/LogVariable.cpp

```diff
@@ -805,6 +805,130 @@ SharedVectorVariableT VectorVariable::toFrequencySpectrum(const SharedVectorVari
     }
 }
 
+SharedVectorVariableT VectorVariable::identifySteadyState(const SteadyStateIdentificationMethodEnumT method, double tol, double win, double stdev, double l1, double l2, double l3)
+{
+    SharedVectorVariableT pRetVar = mpParentLogDataHandler->createOrphanVariable(this->getFullVariableName()+"_ss", TimeDomainType);
+
+    SharedVectorVariableT pTime = this->getSharedTimeOrFrequencyVector();
+    QVector<double> time = pTime->getDataVectorCopy();
+    QVector<double> data = this->getDataVectorCopy();
+
+    if(method == RectangularWindowTest) {
+        int nw = 0;
+        for(int i=0; i<time.size(); ++i) {
+            if(time[i] < win) {
```

Comment:
Is win the start of the window or end of the window, maybe clarify variable names to make code easier to understand

Reply:
`win` is the length of the sliding window. This check makes sure we wait until the entire window is filled with values before we start looking for steady-state. I added a comment block for the function which explains the arguments.